addons/binding/org.openhab.binding.rfxcom/src/main/java/org/openhab/binding/rfxcom/handler/RFXComBridgeHandler.java

```diff
@@ -259,8 +258,7 @@ public void packetReceived(byte[] packet) {
                                 }
 
                                 if (setMode != null) {
-                                    logger.debug("Setting RFXCOM mode using: {}",
-                                            DatatypeConverter.printHexBinary(setMode));
+                                    logger.debug("Setting RFXCOM mode using: {}", HexUtils.bytesToHex(setMode));
```

Comment:
The `HexUtils.bytesToHex()` method is applied every time. Should we encapsulate it by an `if (logger.isDebugEnabled())`?